### PR TITLE
Create crunch42-analysis.yml

### PR DESCRIPTION
**Pull request by bajecogem83**

### CODE REVIEW

File: .github/workflows/crunch42-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  rest-api-static-security-testing:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: 42Crunch REST API Static Security Testing
+        uses: 42Crunch/api-security-audit-action@v1
+        with:
+          # Please create free account at https://platform.42crunch.com/register
+          # Follow these steps to configure API_TOKEN https://docs.42crunch.com/latest/content/tasks/integrate_github_actions.htm
```
Comment (bajecogem83) [inline .github/workflows/crunch42-analysis.yml:41]:
```suggestion
          # Follow these steps to configure API_TOKEN https://docs.42crunch.com/latest/content/tasks/integrate_github_actions.htm
```

**Review by bajecogem83 — COMMENTED** [review]
Ok

**Comment by gbaned** [comment]
@bajecogem83 Can you please sign CLA. Thanks!

**Comment by gbaned** [comment]
@bajecogem83 Any update on this PR? Please. Thanks!

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**